sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SamplesView.java

```diff
@@ -111,8 +111,7 @@ public SamplesView() {
 			fileDownloader.extend(exportButton);
 		}
 
-		if ((isHumanSampleView() && UserProvider.getCurrent().hasUserRight(UserRight.PERFORM_BULK_OPERATIONS_CASE_SAMPLES))
-			|| (isEnvironmentSampleView() && UserProvider.getCurrent().hasUserRight(UserRight.PERFORM_BULK_OPERATIONS))) {
+		if ((isHumanSampleView() || isEnvironmentSampleView()) && UserProvider.getCurrent().hasUserRight(UserRight.PERFORM_BULK_OPERATIONS)) {
```

Comment:
```suggestion
		if (UserProvider.getCurrent().hasUserRight(UserRight.PERFORM_BULK_OPERATIONS)) {
```

The first part of the if clause is no longer necessary since we now have only one bulk operation right for both views